js/mexc.js

```diff
@@ -1586,9 +1586,78 @@ module.exports = class mexc extends Exchange {
         //         ]
         //     }
         //
-        // todo add parsePositions, parsePosition
         const data = this.safeValue (response, 'data', []);
-        return data;
+        return this.parsePositions (data);
+    }
+
+    parsePosition (position, market = undefined) {
+        //
+        //     {
+        //         "positionId": 1394650,
+        //         "symbol": "ETH_USDT",
+        //         "positionType": 1,
+        //         "openType": 1,
+        //         "state": 1,
+        //         "holdVol": 1,
+        //         "frozenVol": 0,
+        //         "closeVol": 0,
+        //         "holdAvgPrice": 1217.3,
+        //         "openAvgPrice": 1217.3,
+        //         "closeAvgPrice": 0,
+        //         "liquidatePrice": 1211.2,
+        //         "oim": 0.1290338,
+        //         "im": 0.1290338,
+        //         "holdFee": 0,
+        //         "realised": -0.0073,
+        //         "leverage": 100,
+        //         "createTime": 1609991676000,
+        //         "updateTime": 1609991676000,
+        //         "autoAddIm": false
+        //     }
+        //
+        market = this.safeMarket (this.safeString (position, 'symbol'));
```

Comment:
Here, the `safeMarket` call is missing a second argument that is the value of the `market` argument of parsePosition. In parsers like parsePosition, parseOrder, parseTrade we should take the market from the arguments and pass that to safeMarket/safeSymbol. This way `safeMarket` can figure the market either from the response structure or from its `market` argument.

Reply:
Okay I will keep this in mind in the future